app/src/main/java/com/kamron/pogoiv/Pokefly.java

```diff
@@ -741,7 +751,23 @@ public void toggleMoreResultsBox() {
 
     @OnClick({R.id.inputAppraisalExpandBox})
     public void toggleAppraisalBox() {
-        toggleVisibility(inputAppraisalExpandBox, appraisalBox);
+        boolean visible = toggleVisibility(inputAppraisalExpandBox, appraisalBox);
+        moveInfoWindow(visible);
+    }
+
+    /**
+     * Move infoLayout window to top/bottom.
+     * @param toTop
+     */
+    private void moveInfoWindow(boolean toTop) {
+        if (toTop) {
+            layoutParams.gravity = Gravity.CENTER | Gravity.TOP;
+            Log.d(TAG, "moveInfoWindow: top");
+        } else {
+            layoutParams.gravity = Gravity.CENTER | Gravity.BOTTOM;
+            Log.d(TAG, "moveInfoWindow: bottom");
+        }
+        windowManager.updateViewLayout(infoLayout, layoutParams);
```

Comment:
There is a lot of space once labels are gone. Below is on Xperia Z5 Compact (smaller then Nexus 5).
![goiv_appraisal_](https://cloud.githubusercontent.com/assets/1045235/18396681/3db0422e-76c5-11e6-84cc-414c65f8325d.jpg)